pom.xml

```diff
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>nexus-plugins</artifactId>
-    <version>3.22.0-01</version>
```

Comment:
version should be 3.22.1-01

Reply:
5b9f64a9f13b2e424dfd8249a779eb86971540d6